protocol.md

```diff
@@ -956,6 +1026,65 @@ or complex data type, e.g.:
       }
     }
     
+### Batch Collection Response
```

Comment:
Won't it be `BatchFinder collection response`?

Reply:
The class name is BatchCollectionResponse.

Reply:
Do you think we should rename it?